projects/rescuechi/petfinder/data_putter.py

```diff
@@ -90,7 +90,7 @@ def append_to_table(engine: sqlalchemy.engine.Engine):
 
 
 def drop_duplicate_rows(engine: sqlalchemy.engine.Engine):
-    """Drops rows with dupliccatd 'id' values.
```

Comment:
Thank you for catching all of my typos 🙈 If only my IDE also had spellcheck for docstrings, lol